.clang-format

```diff
@@ -64,6 +64,7 @@ ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
+TypenameMacros: ['S2N_RESULT']
```

Comment:
While you're at it, you might want to add [S2N_CLEANUP_RESULT](https://github.com/aws/s2n-tls/blob/main/utils/s2n_result.h#L60) so future devs less familiar with clang-format don't run into the same problem.

Reply:
I noticed a few more as well.